Tools/clinic/clinic.py

```diff
@@ -886,9 +886,7 @@ def output_templates(
         converters = [p.converter for p in parameters]
 
         has_option_groups = parameters and (parameters[0].group or parameters[-1].group)
-        default_return_converter = (not f.return_converter or
-            f.return_converter.type == 'PyObject *')
-
+        default_return_converter = f.return_converter.type == 'PyObject *'
```

Comment:
`f.return_converter` is always an instance of `CReturnConverter`, which is always truthy:

https://github.com/python/cpython/blob/ac7605ed197e8b2336d44c8ac8aeae6faa90a768/Tools/clinic/clinic.py#L2540C23-L2540C39